index.html

```diff
@@ -7,6 +7,7 @@
   <title>Build Triage Links for Bugzilla Component(s)</title>
   <link rel="stylesheet" type="text/css" href="jquery-ui-1.11.4/jquery-ui.min.css">
   <link rel="stylesheet" type="text/css" href="index.css">
+  <link rel="icon shortcut" href="Green_bug_and_broom.svg" type="image/svg">
```

Comment:
Since this is intended for Firefox and Bugzilla, I don't think that's a big issue.